projects/components/src/datagrid/details/detail-wrapper.component.html

```diff
@@ -0,0 +1,9 @@
+<clr-dg-row-detail *clrIfExpanded>
+    <ng-template
+        [vcdComponentRendererOutlet]="{
+            rendererSpec: getDetailRenderSpec(),
+            provider: injector
```

Comment:
shouldn't this say:

`injector: injector`

according to https://github.com/vmware/vmware-cloud-director-ui-components/pull/85/files/672bccf0404e7a2ac7d6abd65363b0c006eaf8e1..70a567ee70032312b15d55c4544639b5d5d549ba#diff-7650ed1cc58e90f46bbe8a7229717fbcR39 ?

Reply:
Ok, so when I looked at this comment I was confused because why was this working in the first place if I had such a fundamental thing wrong? So, I removed all references to new Injectors and Providers and as it turns out, when you create a component dynamically the dependency chain is not broken, it's only when you create a component within some dynamic template. So we can remove all references to Injection and Provider and this whole detail wrapping component all together